kura/org.eclipse.kura.api/META-INF/MANIFEST.MF

```diff
@@ -29,13 +29,13 @@ Export-Package: org.eclipse.kura;version="1.3.0",
  org.eclipse.kura.gpio;version="1.0.1",
  org.eclipse.kura.linux.udev;version="1.0.0",
  org.eclipse.kura.message;version="1.1.0",
- org.eclipse.kura.net;version="1.2.1",
+ org.eclipse.kura.net;version="1.2.2",
```

Comment:
I think there is a "misalignment" between historical Kura (i.e a framework mostly targetting ESF use cases) and where Kura is going now. Correct me if I'm wrong but I believe @dwoodard1 and other ETH folks PoV is that while the interface is API, it is *implicitly* meant to be a "@noimplement", i.e people are only expected to consume implementations of this API, not to to provide their own to the framework. This is of course perfectly fine, and there are many cases in the Eclipse platform where that is indeed the case, and where for many reason it is not desirable, or even not supported, to have different implementations of an API.

That being said, the "problem" now that Eclipse Kura is getting more and more adoption, is that those not-to-be-implemented interfaces need to be properly documented (ideally annotations such as @noimplement are used too, to help drive automated tools such as PDE API tooling or bnd), because it is just not safe to assume that people won't want to extend the Kura framework to provide their own "tweaks" - eh, that's what OSGi is great for, right? :-)

Hope this makes sense!